### PR TITLE
Add Semantic Release

### LINKED ISSUE #1: semantic-release into boilerplate
- add semantic release
- use the exec plugin to copy the dist js into boilerplate
- create a branch and PR there

### PR DESCRIPTION
**Pull request by trieloff**
This PR will add a semantic release configuration that will
- create GH branches
- create NPM releases
- update the CHANGELOG.md in GitHub
- create a PR in adobe/helix-project-boilerplate to override the `lib-franklin.js` with the results created in `dist`

you can see an example PR here https://github.com/elc9aya2ls612j/helix-project-boilerplate/pull/4

Fixes #1

### CODE REVIEW

File: publish.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+ORG=elc9aya2ls612j
+REPO=helix-project-boilerplate
+FILENAME=scripts/lib-franklin.js
```
Comment (kptdobe) [inline publish.sh:4]:
```suggestion
FILENAME=scripts/aem.js
```

File: publish.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+ORG=elc9aya2ls612j
```
Comment (kptdobe) [inline publish.sh:2]: I assume there is a good reason but why is the org so weird ?

```suggestion
ORG=adobe
```

**Review by kptdobe — CHANGES_REQUESTED** [review]
Thanks @trieloff. Apart from the file name change, we should merge this PR when we are ready with the first version. Before, we should be able to commit to `main` without triggering a release each time.

File: publish.sh (hunk 2 above)
Comment (trieloff) [reply publish.sh:2]: I've been trying some stuff with IDNs in Helix and still had the org hanging around

**Comment by trieloff** [comment]
I'd rather merge earlier, hope that we don't get accidental releases and dismiss the PRs if we do

**Review by kptdobe — APPROVED** [review]

**Pull request merged**